src/grisp_connect_ws.erl

```diff
@@ -99,10 +99,17 @@ handle_info({gun_ws, Pid, Stream, {text, Text}},
             #state{gun_pid = Pid, ws_stream = Stream} = S) ->
     grisp_connect_client:handle_message(Text),
     {noreply, S};
+handle_info({gun_ws, Pid, Stream, {close, Code, Message}},
+            #state{gun_pid = Pid, ws_stream = Stream} = S) ->
+    ?LOG_WARNING(#{event => stream_closed, code => Code, reason => Message}),
+    {noreply, S};
 handle_info({gun_down, Pid, ws, closed, [Stream]}, #state{gun_pid = Pid, ws_stream = Stream} = S) ->
     ?LOG_WARNING(#{event => ws_closed}),
     grisp_connect_client:disconnected(),
     {noreply, shutdown_gun(S)};
+handle_info({gun_down, Pid, ws, normal, _}, #state{gun_pid = Pid} = S) ->
```

Comment:
How likely is it that it happens? We should add some log here, level depends on how likely we expect it to happen.

Reply:
As I observed, this happens only after a close frame is sent from the server in the stream. Is not an error, just a signal that the WS closed correctly. So I thought that logging the close frame was more then enough, since it is always followed by this `gun_down ws normal `message.

Reply:
Maybe `LOG_INFO`?